public/scripts/main.jsx

```diff
@@ -295,6 +329,25 @@ class MainContent extends React.Component {
 
         </div>
 
+        <ReactTooltip place="bottom" delayShow={700} id="projectTabTooltip">
```

Comment:
Should we make a CesiumTooltip with the default parameters specified, to prevent all of the repetition?